### DISCUSSION TITLE
A history of playback

### DISCUSSION THREAD

**Discussion by mgb744** [LeoKlaus/plappa#112] [Feature Requests]
I have recently moved from android to iOS and have used smart audiobooks player. It had a feature of showing a history of playback, this is really useful especially when I use it in combination the sleep timer and shake to continue playing. It would show when  the book played and paused and sleep timer and each has a time stamp and you are able to go to any of these events.

**Comment by LeoKlaus** [comment]
I added this with the recent 1.4.0 update in TestFlight, it will release to the AppStore with the next update.
